src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__device-photo {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card__device-name {
+  margin-top: 40px;
+
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main-accent);
+}
+
+.card__device-code {
+  margin-top: 4px;
+
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-secondary-accent);
+}
+
+.card__device-price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-top: 24px;
+}
+
+.card__price-caption {
+  display: flex;
+  align-items: flex-end;
+
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-secondary-accent);
+}
+
+.card__total-price {
+  display: flex;
+  align-items: flex-end;
+
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  text-align: right;
+  color: var(--color-main-accent);
+}
+
+.card__button {
+  display: inline-block;
+
+  margin-top: 16px;
+  padding: 12px 70px 12px 69px;
```

Comment:
Let's define the height of the button. It should be 40px.
Also, width of the button should be 100% of parent container